htdocs/class/libraries/vendor/geekwright/regdom/data/public_suffix_list.dat

```diff
@@ -11934,7 +11934,7 @@ shw.io
 // Submitted by Jonathan Rudenberg <jonathan@flynn.io>
 flynnhosting.net
 
-// Forgerock : https://www.forgerock.com
+// Forgerock : https://www.forgerock.com
```

Comment:
For future reference, since this is dynamic data, this change will be overwritten on updates. General rule don't change things in htdocs/class/libraries (unless you are updating the whole set via composer.)

Reply:
OK, got it!